Automattic/wp-calypso
# Performance Profiler: Add Performance Score

Fixes [Performance Profiler: Create performance score section for result page](https://github.com/Automattic/dotcom-forge/issues/8559)

Based on https://github.com/Automattic/wp-calypso/pull/93468

## Proposed Changes

* Create the skeleton for the DashboardContent component.
* Create the PerformanceScore component

## Why are these changes being made?

* This DashboardContent will be used to handle the data while the main Dashboard page will be used to handle URL params and put the pieces together (header, content, footer).
* The PerformanceScore is needed to display the score and its shown on the final layout

## Testing Instructions

* Check this branch on your local machine 
* Go to `http://calypso.localhost:3000/speed-test-tool?url=wordpress.com`
* Check if you see the Performance Score component
* Go to `/calypso/performance-profiler/components/dashboard-content` and change the value on the line `18`
* Refresh the page and check you can see all the 3 states (good, poor, neutral). Ex:  use 45, 60, 95 as values.

| Poor  | Neutral | Good |
| ------------- | ------------- | ------------- |
|![CleanShot 2024-08-12 at 19 17 57@2x](https://github.com/user-attachments/assets/f12ecea0-a676-4ee2-8ca5-c435a74d9edf)|![CleanShot 2024-08-12 at 19 17 43@2x](https://github.com/user-attachments/assets/16fac68b-7b23-4d90-8ca6-a6f6ad5f2fc3)|![CleanShot 2024-08-12 at 19 18 11@2x](https://github.com/user-attachments/assets/9d274525-bde4-4c98-9e0b-6d306540e7b2)
